deepgram/projects.py

```diff
@@ -24,21 +24,17 @@ async def create(self, name: str) -> Project:
             headers={'Content-Type': 'application/json'}
         )
 
-    async def update(self, project_id: str, name: str = None,
-                     company: str = None) -> UpdateResponse:
+    
+    async def update(self, project_id: str, **payload) -> UpdateResponse:
         """Updates a project's information."""
-        payload = {}
-        # there's got to be a better way to do this without non-specific kwargs
-        if name:
-            payload['name'] = name
-        if company:
-            payload['company'] = company
+       
```

Comment:
```suggestion
        payload = {k: v for k, v in {
            'name': name,
            'company': company,
        } if v}
        
```

I think I like the idea of staying explicit about the parameters the endpoint can take, so users don't have to refer back to the docs as much!
We can make it a little more elegant than the previous version, though :p